src/gluonts/nursery/ts_embeddings/ShowEmbeddings.ipynb

```diff
@@ -0,0 +1,158 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
```

Comment:
One of @lostella 's suggestions was to move the notebooks to the examples folder.

Reply:
@jaheba pointed out that jupyter notebooks may not end up installed with the package, so maybe this is not really urgent